crates/astria-conductor-test/kubernetes/deployment.yml

```diff
@@ -132,14 +132,17 @@ spec:
           envFrom:
             - configMapRef:
                 name: metro-env
-        - name: relayer
-          image: "ghcr.io/astriaorg/sequencer-relayer:sha-574f422-debug"
-          command: ["/usr/local/bin/relayer"]
+        - name: astria-sequencer-relayer
+          image: docker.io/library/debian:bookworm-slim
+          command: ["/usr/local/bin/astria-sequencer-relayer"]
           args:
             - "--sequencer-endpoint=http://localhost:1318"
```

Comment:
2/3 of these arguments are default values. `--sequencer-endpoint=http://localhost:1318` is the only one that is not a default because we're manually configuring Metro for 1318, but I think that was changed from 1317 to avoid port conflicts, which are no longer a concern.

Could we start Metro on 1317 again, which would allow us to get rid of all three of these arguments?

Reply:
Yeah, we should totally do this. Let's open an issue though and do it in a follow-up PR to not delay this one.

Maybe we can do this while splitting out the individual containers into separate pods so that we don't have to worry about port overlaps ever again?

Reply:
That sounds like a great idea to me!